src/Symfony/Component/Messenger/Stamp/HandlerArgumentsStamp.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Stamp;
+
+/**
+ * @author Jáchym Toušek <enumag@gmail.com>
+ */
+final class HandlerArgumentsStamp implements NonSendableStampInterface
+{
+    public function __construct(
+        public readonly array $additionalArguments,
```

Comment:
As we are not using public properties in Symfony yet, I think we should make it private and create a proper getter (for consistency with the rest of the code base).